Explicitly specify shfmt indent

jandubois opened the pull request:
It looks like the shfmt v3.12.0 update has broken the .editorconfig lookup in the repo root directory. Specifying the indent explicitly for now.

Upstream issue: https://github.com/mvdan/sh/issues/1173

mook-as commented on bats/Makefile:
```diff
@@ -19,9 +19,9 @@ lint:
-	find scripts -name '*.sh' | xargs shfmt -s -d
+	find tests -name '*.bash' | xargs shfmt -s -d -i 4
+	find tests -name '*.bats' | xargs shfmt -s -d -i 4
+	find scripts -name '*.sh' | xargs shfmt -s -d -i 4
```
Consider using `--indent` instead of `-i`.

jandubois committed:
Explicitly specify shfmt indent

It looks like the shfmt v3.12.0 update has broken the .editorconfig lookup
in the repo root directory. Specifying the indent explicitly for now.

Signed-off-by: Jan Dubois <jan.dubois@suse.com>
